app/javascript/components/widget-display.jsx

```diff
@@ -72,23 +72,19 @@ export const WidgetDisplay = ({
     durationRemaining: null,
   });
 
-  useEffect(() => {
-    dispatch({
-      type: 'startTimer',
-      durationSeconds: widgetTransitionTime,
-    });
-  }, [current.id]);
+  const resetTimer = () => {
+    dispatch({ type: 'startTimer', durationSeconds: widgetTransitionTime });
+  };
 
   useEffect(() => {
     const timerId = setInterval(() => {
       const { durationRemaining, start } = state;
-      if (isPaused(state)) {
+      if (isPaused(state) || !durationRemaining) {
```

Comment:
What does `!durationremaining` mean here? Could we name/extract that as a selector so its self documenting?

Reply:
👍 this is a good idea. I think the idea we want is:

```js
function selectIsTimerRunning(state){
  return !isPaused(state) && state.durationRemaining
}
```